### ISSUE TITLE
[iOS + Android] Inconsistent Log out functionality

### ISSUE THREAD

**Issue by nguyendtt** [trietphamvm/DAR#81]
@tle960 I'm seeing some inconsistency for log out between iOS and Android. For Android, log out option is at the bottom of the side nav vs. in Settings for iOS.  Would you please take a look and recommend a consistent treatment for it? Thanks.

**Comment by tle960** [comment]
Good Catch Trinh!  We would like users to stay logged in and not log out so easily.  As such, we should have the log out option in the settings for both iOS and Android.  In addition, we should have the switch driver to rider or vice versa in the side nav.

In iOS:  please add "log in as driver" in the side nav.
In Andriod:  please move the log out to settings and add "log in as driver" if not already in the side nav.
Please see mocks.
![dar_main_rider_side_nav](https://cloud.githubusercontent.com/assets/5767081/10626441/e92a648e-7763-11e5-9453-1e77d2920cb0.png)
![img_0560](https://cloud.githubusercontent.com/assets/5767081/10626501/8a406dfa-7764-11e5-865a-538edde7861f.PNG)

Please let me know if anyone has any questions.

Thanks.

**Comment by nguyendtt** [comment]
Thank you @tle960 !

**Comment by nguyendtt** [comment]
@cuong-ubesim please assign to @phucdoan  when you're done implementing. Thanks.

**Comment by ThangPk** [comment]
Android updated.
